### DISCUSSION TITLE
Cycle columns from 1 to 10

### DISCUSSION THREAD

**Discussion by danetrata** [gTile/gTile#391] [Ideas]
After little thinking, may be there are other option that can improve your workflow.

Next:
I do not really like how I implemented autotile in columns. I have 10 keyboard shortcut 1 .. 0, pressing them force windows to arrange in corresponding number 1 - 10 of columns. I have feeling that it is awkward and nobody really using it. AT least on my box, several number keys are not working because they are taken by gnome or something else.
I think right way is to do circular arrangments. It should be one key, and pressing it repeatedly will force to autotile in 1,2,3,4, ... columns. Also, I think it should be 2 keyboard shortcuts - one for arranging in columns, one for arranging in rows.
Next, I think that autotile should be first tried on gTile current grid, and windows should be autotiled on a grid on first invocation. On next invocation, number of columns/rows should be changed, but it should be from number of columns/rows in grid.
Also, due to possible large number of max columns/rows (10), it is not convenient to cycle through them. So, it should be two keyboard shourtcuts - one for increasing, one for decreasing number of columns/rows in autotile.

_Originally posted by @scherepanov in https://github.com/gTile/gTile/issues/191#issuecomment-784235397_

**Comment by scherepanov** [comment]
I already forget about all autotile suggestions... thanks for putting them as issues, to keep track
